integration_test/src/test_runner/mod.rs

```diff
@@ -8,6 +10,7 @@ use serde_json::value::Value;
 use std::ptr::null_mut;
 use std::{fmt, fs, path::PathBuf};
 
+use crate::test_runner::test_generator_util::generate_baseline_test_file;
 use thiserror::Error;
 
 const TEST_FILE_DIR: &str = "../resources/integration_test/tests";
```

Comment:
Perhaps because the test_generator is run as an executable vs test, but the directory that works for the test generator in my attempt to run it is `./resources/integration_test/tests`.  Where resources is in the same directory and not parent dir.